builtin/settingtypes.txt

```diff
@@ -903,8 +903,12 @@ default_privs (Default privileges) string interact, shout
 #    Privileges that players with basic_privs can grant
 basic_privs (Basic privileges) string interact, shout
 
-#    If enabled, disable cheat prevention in multiplayer.
-disable_anticheat (Disable anticheat) bool false
+#    Server anticheat configuration
+anticheat_flags (Anticheat flags) flags digging,interaction,movement digging,interaction,movement
+
+#    Tolerance of movement anticheat detector.
+#    Increase the value if players experience stuttery movement.
+anticheat_movement_tolerance (Anticheat movement tolerance) float 1.0 1.0
```

Comment:
- It should be clarified that the setting is positive, i.e. each flag enables a part of the anticheat code.

- "movement ~~anti~~cheat detector" - it's not about detecting anticheating, it's about detecting cheating